roro/auth.py

```diff
@@ -64,11 +64,15 @@ def __init__(self, file=None):
         # work-around to fix that
         if file is None:
             file = self.find_default_file()
-        super().__init__(file=file)
+
+        if PY2:
+            _netrc.__init__(self, file=file)
+        else:
+            super().__init__(file=file)
```

Comment:
Let us just the first way. No need for PY2 condition.

Reply:
Okay